packages/components/src/components/audio/audio.stories.ts

```diff
@@ -0,0 +1,366 @@
+import '../../solid-components';
+import { html } from 'lit-html';
+import { storybookDefaults, storybookTemplate } from '../../../scripts/storybook/helper';
+import { withActions } from '@storybook/addon-actions/decorator';
+import type { ConstantDefinition } from '../../../scripts/storybook/helper';
+
+const { argTypes, args, parameters } = storybookDefaults('sd-audio');
+const { generateTemplate } = storybookTemplate('sd-audio');
+
+export default {
+  title: 'Components/sd-audio',
+  component: 'sd-audio',
+  args,
+  argTypes,
+  parameters: { ...parameters },
+  decorators: [withActions] as any
+};
+
+const audioElementScript = html`<script>
```

Comment:
do we really need all of that code in Storybook? looks like our component is not able to be used without all of that... :/

![image](https://github.com/user-attachments/assets/af9c2f9b-6e7c-475f-bb6e-6cd5567612dd)

Reply:
Good point. My expectation would be, that default HTML audio would be wired up automatically.